targets/bcm27xx/profiles/bcm2711/diffconfig

```diff
@@ -0,0 +1,4 @@
+CONFIG_TARGET_bcm27xx=y
```

Comment:
A few diffconfigs included, is this intentional? I think they should be removed for this PR?

Reply:
Including the diffconfigs was unintentional - they should be removed (will do this when I'm back from travels late this week).